.github/CONTRIBUTING.md

```diff
@@ -540,7 +540,113 @@ For more on good git commits, see (among _many_ other):
+
+Should you find yourself in that situation and want to salvage as much of your work as possible: create an issue following [the process for contribution](#the-process-for-contribution) and make changes to your pull-request as necessary.
+
+### The good pull-request has a title worthy of the changelog
+
+Besides informing what the changes in the pull-request does - the title of your pull-request will be used in [the changelog](https://cookbook.kirby.design/home/changelog) to represent your changes! 
+![](images/issue-title-to-changelog.png)
+
+So make the title descriptive - after writing the title, ask yourself if you would feel informed what the changes in the PR entail if you read it as part of a changelog. 
+
+[keepachangelog.com](https://keepachangelog.com) has a neat list of good verbs to start out with, you can use as inspiration for your title:
+
+* `Added` for new features
+* `Changed` for changes in existing functionality 
+* `Deprecated` for soon-to-be removed features 
+* `Removed` for now removed features 
+* `Fixed` for any bug fixes 
+
+Example titles that are worthy of the changelog: 
+
+* "Added default styling for text links"
+* "Removed kirby-toolbar"
+* "Fixed kirby-radio-group TypeError"
+
+### The good pull-request uses the template
+
+As with writing [good issues](#issue), do not ignore the template text when creating a pull-request. 
+
+It provides a checklist to help you follow [the process of contributing](#the-process-of-contributing), ease collaboration by providing structure and reminds you to provide information the reviewer might be looking for. 
+
+Filling out the following part of the template properly, will cause the related issue to be closed automatically once the pull-request has been merged: 
+
+> This PR closes # (reference issue number here)
+
+For example: 
+
+> This PR closes [#1427](https://github.com/kirbydesign/designsystem/issues/1427)
+
+If the pull-request solves more than one issue also mention these such that they also are closed automatically: 
+
+> This PR closes [#1427](https://github.com/kirbydesign/designsystem/issues/1427), closes [#1428](https://github.com/kirbydesign/designsystem/issues/1428), closes [#1429](https://github.com/kirbydesign/designsystem/issues/1429)
+
+### The good pull-request has a logical commit history
+
+Maintaining a logical commit history constructed using [good commits](#commit) helps the reviewer easily get an overview and understanding of the changes contained in a pull-request.
+
+If you are having trouble maintaining a clean commit history, you could eventually try writing [pre-emptive commit comments](https://arialdomartini.wordpress.com/2012/09/03/pre-emptive-commit-comments/). This consists of writing down the commit comment, before writing your code. This can help you stay focused on a single task at a time. 
```

Comment:
Maybe skip the word "eventually" or rephrase to "[...] you could consider writing [pre-emptive commit comments](https://arialdomartini.wordpress.com/2012/09/03/pre-emptive-commit-comments/)"